pkg/ctl/enable/profile.go

```diff
@@ -57,14 +58,15 @@ func enableProfileCmd(cmd *cmdutils.Cmd) {
 	})
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
-		fs.StringVarP(&opts.quickstartNameArg, "name", "", "", "name or URL of the Quick Start profile. For example, app-dev.")
+		fs.StringVarP(&opts.profileName, "name", "", "", "name or URL of the Quick Start profile. For example, app-dev.")
+		fs.StringVarP(&opts.profileRevision, "revision", "", "master", "revision of the Quick Start profile.")
```

Comment:
Please remove this new flag and use `--git-branch` from line 64.

Reply:
The revision is for the profile repo not the destination one. Please see #1436 on why I've added this.